synapse/events/utils.py

```diff
@@ -32,13 +32,13 @@
 from synapse.api.constants import EventContentFields, EventTypes, RelationTypes
 from synapse.api.errors import Codes, SynapseError
 from synapse.api.room_versions import RoomVersion
+from synapse.handlers.relations import BundledAggregations
```

Comment:
is this deliberate?

I'm thinking that the import was probably conditional for a reason; and that in any case we don't really want to add dependencies on `synapse.handlers` from `synapse.events`.

Reply:
I think an older version required this, but definitely not needed now. 👍